### DISCUSSION TITLE
I'm having trouble with this in Homebridge

### DISCUSSION THREAD

**Discussion by jrushford** [Sunoo/homebridge-camera-ffmpeg#1439] [Q&A] [answered]
I'm trying to get an Amcrest camera model IP5M-B1276EW-AI working with version 3.1.4 of this plugin with Homebridge.  I've been trying different settings but cannot get the camera to stream properly with this plugin.  Using debug, this is the Homebridge log from my attempt to access the camera in the Home App:

[homebridge.log](https://github.com/Sunoo/homebridge-camera-ffmpeg/files/14056087/homebridge.log)

This is my current home bridge config:

[config.json](https://github.com/Sunoo/homebridge-camera-ffmpeg/files/14056099/config.json)

I've got several other of these cameras I've been trying to add and I get the same errors for them.  The source URL works fine in VLC which I've used for verifying.  VLC used a "H264 - MPEG-4 AVC (part 10)(h264) codec for video and "MPEG AAC Audio (mp4a).  I see no problems with this camera using VLC just this Homebridge plugin.

I did have it working a few months back with an earlier release of this plugin but my home bridge went belly up in my TrueNAS scale and I've had to reinstall Homebridge on a Debian VM as the Docker app has issues in the current release of TrueNAS scale.

Anyway, I'm hoping someone here can help met to get the plugin working with this camera.  

thanks for your time
John Rushford
jjrushford@gmail.com

**Comment by Sunoo** [comment]
ENOENT means ffmpeg-for-homebridge didn’t get installed for some reason. This can often be fixed with a reinstall of the plugin.

**Reply by jrushford** [reply, marked as answer]
I've got it fixed.  I overlooked installing ffmpeg on the Debian vm.  Thanks for your reply
